scss/components/_skip-controls.scss

```diff
@@ -0,0 +1,80 @@
+
+.oo-skip-controls {
+  $oo-skip-controls-btn-size: 1.52em;
+  @include flex-center-children(row);
+  @include absolute-center;
+  font-size: 14px;
+  pointer-events: none;
+  transition: 0.2s opacity ease-in-out, 0.3s transform ease-in-out;
+  will-change: transform, opacity;
+
+  &.oo-inactive {
+    opacity: 0;
+    transform: translate(-50%, -90%);
+  }
+
+  &.oo-in-background {
+    opacity: 0.5;
+  }
+
+  .oo-focusable-btn {
+    position: relative;
+    width: $oo-skip-controls-btn-size;
+    height: $oo-skip-controls-btn-size;
+    margin: 0 10px;
+    padding: 0;
+    border: none;
+    background: none;
+    font-size: 1.7em;
+    pointer-events: auto;
+  }
+
+  .oo-focusable-btn:disabled {
+    opacity: 0.4;
+  }
+
+  .oo-icon {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  .oo-center-button .oo-icon {
+    font-size: 1.4em;
+  }
+
+  .oo-btn-counter {
+    @include flex-center-children(row);
+    color: #fff;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    font-size: 0.55em;
+    text-align: center;
+    box-sizing: border-box;
+    backface-visibility: hidden;
```

Comment:
Why does this line have a red background? Do you see the same?

Reply:
I think it's because `backface-visibility` is an experimental technology. This can be considered as progressive enhancement though, browsers that don't support this (or deprecate it) will just ignore it. This is needed in order to prevent the button text from flickering during animations

Reply:
Sounds good